services/business/checkers/virtual_service_checker.go

```diff
@@ -50,6 +51,14 @@ func (in VirtualServiceChecker) runGroupChecks() models.IstioValidations {
 		validations = validations.MergeValidations(checker.Check())
 	}
 
+	enabledDRCheckers := []GroupChecker{
```

Comment:
IMO, I would say that this validation takes `DestinationRules` as subject so I would place it into the DestinationRule checker.

Reply:
Yep, moving.